src/test/java/org/fundacion/jala/converter/core/ExtractMetadataTest.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacion.jala.converter.core;
```

Comment:
Copyright doc is missing in all classes

Reply:
We agreed that javadoc and copyright would not be put in the tests